mountpoint-s3/scripts/fs_bench.sh

```diff
@@ -51,13 +52,23 @@ run_fio_job() {
   for i in $(seq 1 $iterations);
   do
     echo -n "${i};"
-    fio --thread \
+    # we know each fio job will be running for exactly 1 minute from the job definitions.
+    # there must be something wrong if it takes longer than that.
+    set +e
+    timeout 300s fio --thread \
       --output=${results_dir}/${job_name}_iter${i}.json \
       --output-format=json \
       --directory=${mount_dir} \
       --filename=${bench_file} \
       --eta=never \
       ${job_file}
+    job_status=$?
+    set -e
+    if [ $job_status -ne 0 ]; then
+      tail -1000 ${log_dir}/mountpoint-s3-*
```

Comment:
I'm not familiar with that syntax for tail, but presumably it works.

If we do another revision of this thread, I'd recommend using `-n <NUM>`.

Reply:
ah, that might be better. I always use it this way, so I'm quite sure it works.